lang/c/c.py

```diff
@@ -188,7 +188,7 @@ def c_cast(m) -> str:
 
 
 @mod.capture(rule="[<self.stdint_signed>] <self.stdint_types> [<self.c_pointers>+]")
-def c_stdint_cast(m) -> str:
+def stdint_cast(m) -> str:
```

Comment:
I wonder whether we should change the `.talon` file to match this instead.

Reply:
My reasoning for changing the ```.py``` file instead was that this happened to be the only place where the 'c_' prefix was used in front of a 'stdint' name. Changing the ```.talon``` file to match this one difference could potentially result in stylistic inconsistency.